locustfile.py

```diff
@@ -10,13 +10,14 @@
 from locust import HttpLocust, TaskSet, task
 from hashlib import sha1
 from random import randint
+import random
+import string
 
 HOST = "http://localhost:8080"
```

Comment:
I had an issue running this as is because my host was set to `french.internal.azavea.com`. Now that the host is dynamic (based on `.env`), we could just accept it as a command line argument to the script and update the docstring accordingly:
```
locust --host=http://localhost:8080
```